src/libs/ReceiptUtils.js

```diff
@@ -38,18 +38,18 @@ function validateReceipt(file) {
  * @returns {Object}
  */
 function getThumbnailAndImageURIs(path, filename) {
+    const isReceiptImage = Str.isImage(filename);
+
     // For local files, we won't have a thumbnail yet
-    if (path.startsWith('blob:') || path.startsWith('file:')) {
+    if (isReceiptImage && (path.startsWith('blob:') || path.startsWith('file:'))) {
```

Comment:
Logically, the `&&` has higher precedence here, meaning this applies for images that start with `blob:` or any other file, right?  I'm not sure if that's the intended behavior or if what we want is any image that starts with `blob:` or `file:` to return here.

Reply:
This should apply for receipts that are image AND start with blob or file, no?

Reply:
Right now, this applies for images that start with "blob", or anything that starts with "file".

<img width="179" alt="Screenshot 2023-08-28 at 17 26 10" src="https://github.com/Expensify/App/assets/705000/c5aa11d5-76a6-493a-b178-669c001a1e37">

I think what you might want is `image && (blob || file)`, but I'm not actually sure because I'm don't really understand the fix.

Reply:
I'm confused, isn't `image && (blob || file)` what I have here?

Reply:
Maybe I misread this. It definitely looks like this now.